src/pages/Jam.svelte

```diff
@@ -94,30 +94,35 @@
             requestAnimationFrame(updatePitch);
             return;
         }
-        let noteNum = noteFromPitch(loudestFreq, octaveLength);
-        if (noteNum !== noteHistory[noteHistory.length - 1]) {
```

Comment:
This chunk is really just inverting the condition to return early and remove a level of indentation